examples/marvel/lib/src/screens/home.dart

```diff
@@ -89,10 +89,14 @@ class Home extends HookConsumerWidget {
             child: const Center(child: CircularProgressIndicator()),
           ),
           error: (err, stack) {
+            String? errorMessage;
+            if (err is DioException) {
+              errorMessage = err.message;
+            }
             return Scaffold(
               appBar: AppBar(title: const Text('Error')),
               body: Center(
-                child: Text('$err'),
+                child: Text(errorMessage ?? '$err'),
```

Comment:
Use Dart 3's switch instead. This would be more readable

Reply:
Thank you for your review. Dart 3's switch is used now, Remi.